apps/mull-api/src/app/entities/post.entity.ts

```diff
@@ -13,30 +14,38 @@ import { PostReaction } from './post-reaction.entity';
 import { User } from './user.entity';
 
 @Entity()
+@ObjectType()
 export class Post {
+  @Field(() => Int)
   @PrimaryGeneratedColumn()
   id: number;
 
-  @OneToOne(() => User)
-  @JoinColumn()
+  @Field(/* istanbul ignore next */ () => User)
```

Comment:
💡 You added `/* istanbul ignore next */` to these two lines, but not to all the other places where you added anon function in the entities. Might be good to also add it everywhere else.